server/issue.go

```diff
@@ -791,6 +791,9 @@ func (s *Server) createPipelineFromIssue(ctx context.Context, issueCreate *api.I
 
 		// Tenant mode project pipeline has its own generation.
 		if project.TenantMode == api.TenantModeTenant {
+			if !s.feature(api.FeatureMultiTenancy) {
```

Comment:
We need it on line 671 too for issueCreate.Type == api.IssueDatabaseCreate.

Reply:
Also a few other places such as moving database from one project to another. We can do a reference search on api.TenantModeTenant.

Reply:
Thanks for remind, updated now.